index.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
```

Comment:
- [ ] Nice working on your Index page, Kindly check the link to the style sheet, the page is not loading properly, the stylesheet is in the style folder, and the link should be `style/style.css`